op-proposer/proposer/abi_test.go

```diff
@@ -19,6 +19,9 @@ import (
 // setupL2OutputOracle deploys the L2 Output Oracle contract to a simulated backend
 func setupL2OutputOracle() (common.Address, *bind.TransactOpts, *backends.SimulatedBackend, *bindings.L2OutputOracle, error) {
 	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		return common.Address{}, nil, nil, nil, err
+	}
 	from := crypto.PubkeyToAddress(privateKey.PublicKey)
 	if err != nil {
 		return common.Address{}, nil, nil, nil, err
```

Comment:
you can now remove the bottom `if err != nil {` check

Reply:
ah, I see, this was a mis-placed err check. I'll probably port this over by itself